dashboard/app/controllers/registrations_controller.rb

```diff
@@ -59,6 +59,11 @@ def begin_sign_up
 
     if @user.errors.blank?
       PartialRegistration.persist_attributes(session, @user)
+    else
+      # Currently, the only error we're checking for is a duplicate email
```

Comment:
Thank you for leaving a comment here about this! That's interesting that this is the only thing we're checking for with `@user.errors`